docs/markdown/howtox.md

```diff
@@ -155,9 +155,6 @@ $ ninja coverage-html (or coverage-xml)
 
 The coverage report can be found in the meson-logs subdirectory.
 
-Note: Currently, Meson does not support generating coverage reports
-with Clang.
-
```

Comment:
Instead of removing this comment altogether, it should be replaced with something like
```markdown
*(new in 0.55.0)* llvm-cov suppoprt for use with clang
```